### ISSUE TITLE
Staging Accessibility Review Findings for AMA/Decision Review Follow-on (#7521)

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7647]
**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

This epic will contain the VSA accessibility staging review findings issue tickets.

## Point of Contact

**VFS Point of Contact:** Jennifer

**Comment by 1Copenut** [comment]
Good finds @jenstrickland . https://github.com/department-of-veterans-affairs/va.gov-team/issues/7769 will be a breaker in the CI/CD pipeline axe check, so fixing it should be a near-term item if possible.

**Comment by jenstrickland** [comment]
@MickinSahni All issues are now noted in this epic. 

Next step, I'll inventory the issues in a spreadsheet, and then remove the vsa-public-websites from those that are for VSP.  I'll @ you here when that is complete.

**Comment by jenstrickland** [comment]
@MickinSahni All of the issues above are now treated in two ways:
1. If the issue begins with [WORDS], then it is assigned to vsa-public-websites.
1. If the issue begins with words before [WORDS], then it is assigned to sitewide teams, and your team label is removed.

Your team's issues are listed on the [VA 508 Product Review List](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-product-review-list.md#appeal-modernization-process-ama-phase-2).

**Comment by FranECross** [comment]
@jilladams The tickets on this epic have the CMS team noted, but the epic has PW. Shouldn't the epic also be CMS team? :) Thanks

**Comment by jilladams** [comment]
Checked the tickets in epic - in all cases, Public Websites was originally listed, and then removed, so I agree and have done the same thing here.
